src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/CatchParameterNameCheck.java

```diff
@@ -61,7 +61,26 @@
  *   &lt;property name="format" value="^[a-z][a-zA-Z0-9]+$"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>
+ *   The following example shows that how the check validates the parameter
+ *   names with a lower case letter, followed by any letters or digits:
+ * </p>
+ * <pre>
+ * &lt;module name="CatchParameterName"&gt;
+ *   &lt;property name="format" value="^[a-z][a-zA-Z]+$"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class Test {
+ *   public void test1(String s1, int num, char[] charArray) { // OK
+ *     ...
+ *   }
+ *   public void test2(int Number) { //ERROR
```

Comment:
How do I use the "cat" command in windows? I found some error in my case. I think I need to pull request again, right? Because the words ". . ." could not be compilable in javac. I think I need to get it with comment line.

Reply:
> I think I need to pull request again, right? 

No. See my comment at https://github.com/checkstyle/checkstyle/pull/6063#issuecomment-411907254 .